### ISSUE TITLE
Documentation 2022a

### ISSUE THREAD

**Issue by JasonGlazer** [NREL/EnergyPlus#9458]
Issue overview
--------------
A place to keep an ongoing list of documentation issues.

**Comment by JasonGlazer** [comment]
- [x] In the IOref under Output:Variable, the schedule value option is not described. To get the hourly schedule values for a schedule named OCCUPY-1 the following syntax works:

Output:Variable,OCCUPY-1,schedule value,hourly;

and wild cards work too:

 Output:Variable,*,schedule value,hourly;

will add hourly values for all schedules to the ESO, CSV, output JSON, (probably) SQLite

**Comment by mjwitte** [comment]
Output:Variable "Schedule Value" is described under Schedules, Outputs, v22.1 I/O Ref pdf p. 149.

**Comment by JasonGlazer** [comment]
> Output:Variable "Schedule Value" is described under Schedules, Outputs, v22.1 I/O Ref pdf p. 149.

Great, maybe we just need a reference from Output:Variable back to that.

**Comment by jcyuan2020** [comment]
- [x] A residual item in the last batch of documentation issues/fixes: https://github.com/NREL/EnergyPlus/issues/9111#issuecomment-1086682009

- [x] Re-organize the heading structures of sections of "Space Thermal Output(s)" and "Zone Thermal Output(s)".

**Comment by eringold** [comment]
- [x] IOref for ZoneHVAC:WaterToAirHeatPump, field: Supply Air Fan Object Type states "Fan type must be Fan:OnOff": https://github.com/NREL/EnergyPlus/blob/a937bf2b63894e1d38389c213fbaf5dec6d97cb5/doc/input-output-reference/src/overview/group-zone-forced-air-units.tex#L3045

Should also include Fan:SystemModel:
```
   A7,  \field Supply Air Fan Object Type
        \required-field
        \type choice
        \key Fan:SystemModel
        \key Fan:OnOff
```

**Comment by jcyuan2020** [comment]
- [x] Section 1.9.18.1.2, Page 177 in IORef (v22.1), incomplete sentence:  
![image](https://user-images.githubusercontent.com/64040111/173871947-8dfe4e25-8f1b-44c3-af36-3024e38e09be.png)

Maybe the "In this case," at the end should be placed to the beginning of the second sentence in this paragraph.

**Comment by jcyuan2020** [comment]
- [x] In IORef, the correct example file name in the following text should be "CmplxGlz_SmOff_IntExtShading.idf" instead: 

![image](https://user-images.githubusercontent.com/64040111/174675650-3332fd19-fc3a-476d-a987-23f6e0c19334.png)

- [x] A general sweep fixes/improvement across the documentation in the surface construction elements group, such as the moisture model and the window shade sections.

**Comment by mitchute** [comment]
- [x] The `ZoneAirMassFlowConservation` example should reflect the options that are currently available.

![Screen Shot 2022-06-28 at 6 25 17 PM](https://user-images.githubusercontent.com/2985979/176326034-a1d10603-9f9b-4cd5-813c-4dff4aabd397.png)

![Screen Shot 2022-06-28 at 6 25 03 PM](https://user-images.githubusercontent.com/2985979/176326040-11761afd-9f3a-4ae7-863a-a23c1017cc58.png)

**Comment by jcyuan2020** [comment]
- [x] Coil:WaterHeating:AirToWaterHeatPump:VariableSpeed section missing a level of heading for "Inputs":

![image](https://user-images.githubusercontent.com/64040111/184411439-b7f526e5-09a7-4aa7-82b7-f65715e287eb.png)

- [x] A few more similar occurrences like the above ones across a few different tex files are also to be fixed. In general, fixing these heading levels would help reduce the number of (unnecessary) bookmarks in the final PDF generated.

**Comment by jcyuan2020** [comment]
- [x] Infiltration mass flow rate output variables should be of type "average" instead of "sum":
![image](https://user-images.githubusercontent.com/64040111/184412651-c2f3e3a0-68c5-450e-a398-95a036bb5dbe.png)

**Comment by jcyuan2020** [comment]
- [x] A few empty and/or duplicated labels caused LaTeX showing compile warnings:

![image](https://user-images.githubusercontent.com/64040111/187293348-ff5c4838-5871-460b-a4c8-e6cc93ed5ef2.png)

![image](https://user-images.githubusercontent.com/64040111/187293134-534e0ea3-5ec9-4e26-a99e-da63ec82f1aa.png)

**Comment by nealkruis** [comment]
@jcyuan2020 these are related to the addition of the ASHRAE 205 chiller object. @tanaya-bigladder was going to clean them up on a follow up branch, but if you fix them we can just merge your changes in. Up to you how you'd like to handle it.

**Comment by jcyuan2020** [comment]
@nealkruis @tanaya-bigladder Thanks! Minor issue. I've got them fixed in a branch for the documentation issues.

**Comment by jcyuan2020** [comment]
- [x] Figure caption text got separated off from the figure environment (IORef v22.1, Page 1993):
![image](https://user-images.githubusercontent.com/64040111/187960748-a337ceb8-e8bd-4b8b-864a-fcf86ff81cfe.png)

- [x] And another one of the same issue above in Page 1994.

**Comment by jcyuan2020** [comment]
- [x] IORef v22.1, Page 158: 24 hrs should be 24*3600= 86400s, instead of 87600s:
![image](https://user-images.githubusercontent.com/64040111/187963177-c484ec8a-fbfc-4aa9-8757-fd82599e27ad.png)

**Comment by jcyuan2020** [comment]
- [x]  The default value for AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl field "Minimum Outdoor Air Temperature in Cooling Mode" got contradicting information:
![image](https://user-images.githubusercontent.com/64040111/187970140-01561b05-6413-4454-92d0-3032847a1ab4.png)
The correct value should be `-6C` according to the IDD information.

- [x] The next field named "Maximum Outdoor Air Temperature in Cooling Mode" also has some incorrect information:
![image](https://user-images.githubusercontent.com/64040111/187970210-5b032cac-2bf2-48bc-96f6-81a1b8432ea2.png)
According to IDD, this value should be `43C`.

- [x] Another field named "Outdoor Unit Reference Subcooling":
![image](https://user-images.githubusercontent.com/64040111/187972413-fafe04a1-418b-46a5-9229-4d4ce8031125.png)
According to IDD, this value should be `5C`. 

- [x] There is a default value of 5m in the IDD for the field named "Height Difference Between Outdoor Unit and Indoor Units":
![image](https://user-images.githubusercontent.com/64040111/187976712-469c611a-5117-47b6-8e00-ff6e22ac2389.png)
The default value description should be added for this field.

**Comment by jcyuan2020** [comment]
- [x] IORef v22.1 Page 517, near the bottom of the page:
![image](https://user-images.githubusercontent.com/64040111/189579457-b86c9b57-f960-4eb5-bda6-407fa16f2d4d.png)

There is no "Figure 51" in the document. According to the context, this should be pointed to Figure 1.72.

**Comment by mjwitte** [comment]
@jcyuan2020 Can this defect be closed?

**Comment by jcyuan2020** [comment]
@mjwitte Yes. All items have been addressed.

**Comment by mjwitte** [comment]
Closed via #9629
